src/main/java/org/cryptomator/ui/controls/FontAwesome5StatusView.java

```diff
@@ -0,0 +1,69 @@
+package org.cryptomator.ui.controls;
+
+import com.tobiasdiez.easybind.EasyBind;
+
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.scene.control.ContentDisplay;
+import javafx.scene.control.Label;
+
+/**
+ * Similar to the {@link FontAwesome5IconView}, except that if the spinner glyph is selected, an animated facsimile is used.
+ */
+public class FontAwesome5StatusView extends Label {
```

Comment:
why is this a label?

Reply:
because the orignial intention was to use it in any context where a label can be used. But i'll remove it.

Reply:
Changed to StackPane in 58eb6a6